src/main/java/org/mcteam/ancientgates/util/BlockUtil.java

```diff
@@ -28,7 +28,7 @@ public class BlockUtil {
 		standableMaterials = new EnumMap<>(Material.class);
 		try {
 			Arrays.asList(GateMaterial.values()).stream().forEach(x -> standableMaterials.put(x.getMaterial(), true));
-			standableMaterials.put(Material.AIR, true); // 0 Air
+			standableMaterials.put(XMaterial.AIR.parseMaterial(), true); // 0 Air
```

Comment:
I believe it would be necessary to add CAVE_AIR to the standable materials as well, not entirely sure if it would make a difference.